webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,17 @@ function BugForm() {
     }, this));
   };
 
+  // See function autoExpand in issues.js
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand",function(){})
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.style.overflow = 'hidden';
+        this.style.height = 0;
```

Comment:
Why are we setting height to 0, then resetting it to `scrollHeight` right after? It's the `scrollHeight` we're really interested in.